projects/components/src/list-view/list-view.component.ts

```diff
@@ -31,6 +46,25 @@ export class ListViewComponent {
 
   @Input()
   public records?: ListViewRecord[];
+
+  @Input()
+  public actionType?: ListViewActionType;
+
+  private readonly scopeQueryParam: string = 'endpoint-traces';
+  private readonly filterBuilder: StringMapFilterBuilder = new StringMapFilterBuilder();
+
+  public constructor(private readonly navigationService: NavigationService) {}
+
+  public searchAttributes(key: string, value: string): void {
+    this.navigationService.navigate({
```

Comment:
this should use the logic from explorer service as mentioned on the other PR.

Reply:
Used the same